core/server/worker/pom.xml

```diff
@@ -109,7 +109,10 @@
       <artifactId>hamcrest</artifactId>
       <scope>test</scope>
     </dependency>
-
```

Comment:
keep a new line between the last external test dependency and the internal test dependency comment

Reply:
Fixed